src/test/java/teammates/ui/webapi/UpdateFeedbackSessionActionTest.java

```diff
@@ -37,17 +41,18 @@ protected void testExecute() {
 
         ______TS("Not enough parameters");
 
-        verifyHttpParameterFailure();
```

Comment:
I think we can leave this case here (failure if no body or params provided).

Reply:
I think I initially replaced all of them because the following (at line 48) would fail because `InvalidHttpRequestBodyException` would be thrown first before `InvalidHttpParameterException` (based on the ordering in `UpdateFeedbackSessionAction`).

```java
verifyHttpParameterFailure(updateRequest, Const.ParamsNames.FEEDBACK_SESSION_NAME, session.getFeedbackSessionName());
```

Looking at it again now, the ordering (of getting the course ID param before the request body in `UpdateFeedbackSessionAction`) would mean that `verifyHttpParameterFailure()` would pass so it could have instead not been replaced like you suggested. However, I think since both no body and no params would be invalid inputs, I think it may be better if I check each invalid input on its own. So I would have to add `verifyHttpRequestBodyFailure` instead to check no body on its own while the others check incomplete params or the lack of params.